### PR TITLE
Small change v4 lavalink part 1.2

### PR DESCRIPTION
**Pull request by lavecat**
Change of self._is_connected =res ["voice"]["connected"] to self._is_connected = res["state"]["connected"] and self._ping = res["state"]["ping"] | change of payload["track"]["encoded"] in WS 
--
add of prossing_single_track to loadtype = 'track' ( for a futur pull request )

### CODE REVIEW

File: lavaplay/utlits.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def prossing_tracks(tracks: list) -> t.List[Track]:
         info = track["info"]
         list_tracks.append(
             Track(
-                track=track["track"],
+                encoded=track["encoded"],  # Use 'encoded' instead of 'track'
```
Comment (HazemMeqdad) [inline lavaplay/utlits.py:33]: Could you please remove this comment so it is not needed

File: lavaplay/ws.py [hunk 2]
```diff
@@ -145,8 +145,8 @@ async def callback(self, payload: dict):
             
     async def event_dispatch(self, payload: dict):
         # encodedTrack is for Lavalink new version v4
-        if payload.get("encodedTrack"):
-            track = await self.node.decodetrack(payload.get("encodedTrack"))
+        if payload["track"]["encoded"]:
+            track = await self.node.decodetrack(payload["track"]["encoded"])
```
Comment (HazemMeqdad) [inline lavaplay/ws.py:149]: The same point of use .get

File: lavaplay/utlits.py [hunk 3]
```diff
@@ -39,7 +39,35 @@ def prossing_tracks(tracks: list) -> t.List[Track]:
                 position=info["position"],
                 source_name=info.get("sourceName", None),
                 title=info.get("title", None),
-                uri=info["uri"]
+                uri=info["uri"],
+                artworkUrl=info["artworkUrl"],
+                isrc=info["isrc"]
```
Comment (HazemMeqdad) [inline lavaplay/utlits.py:44]: Use here info.get() it is safeist

File: lavaplay/node_manager.py [hunk 4]
```diff
@@ -231,17 +231,20 @@ async def decodetrack(self, track: str) -> Track:
...
+            is_seekable=info["isSeekable"],
+            author=info["author"],
+            length=info["length"],
+            is_stream=info["isStream"],
+            position=info["position"],
+            source_name=info.get("sourceName", None),
+            title=info.get("title", None),
+            uri=info["uri"],
+            artworkUrl=info["artworkUrl"],
+            isrc=info["isrc"]
```
Comment (HazemMeqdad) [inline lavaplay/node_manager.py:247]: same change to use .get

**Review by HazemMeqdad — CHANGES_REQUESTED** [review]

File: lavaplay/ws.py (hunk 2 above)
Comment (lavecat) [reply lavaplay/ws.py]: good

File: lavaplay/node_manager.py (hunk 4 above)
Comment (lavecat) [reply lavaplay/node_manager.py]: good

File: lavaplay/utlits.py (hunk 1 above)
Comment (lavecat) [reply lavaplay/utlits.py]: good

File: lavaplay/utlits.py (hunk 3 above)
Comment (lavecat) [reply lavaplay/utlits.py]: good

**Review by HazemMeqdad — APPROVED** [review]
LGTM, thank you.

**Pull request merged**